source/context_menus/_context_menu_stats.js

```diff
@@ -0,0 +1,58 @@
+const { PermissionFlagBits, InteractionContextType } = require('discord.js');
+const { UserContextMenuWrapper } = require('../classes');
+
+const mainId = "User Stats";
```

Comment:
Please update `wiki/Developers.md` with the context for this exception for id.

Reply:
Should be up to date now!